cmd/keytransparency-sequencer/main.go

```diff
@@ -228,4 +228,13 @@ func runSequencer(ctx context.Context, conn *grpc.ClientConn,
 			glog.Errorf("PeriodicallyRun(RunBatchForAllMasterships): %v", err)
 		}
 	})
+
+	sequencer.PeriodicallyRun(ctx, time.Tick(*refresh), func(ctx context.Context) {
```

Comment:
Need to use a goroutine above, since sequencer.PeriodicallyRun runs in a loop until context cancellation.